tests/src/e2e.spec.ts

```diff
@@ -81,7 +111,36 @@ test.skip('Check welcome page is displayed and that we are redirected to the mai
   // check we have the dashboard page
   const dashboardTitle = page.getByRole('heading', { name: 'Dashboard' });
   await playExpect(dashboardTitle).toBeVisible();
+});
 
-  expect(windowState.isCrashed, 'The app has crashed').toBeFalsy();
-  expect(windowState.isVisible, 'The main window was not visible').toBeTruthy();
+test('Verify main UI elements are present - Application tray', async () => {
+  const appTray = page.locator('.place-self-end');
```

Comment:
It looks like there are issues if you start to use such locators

For example current locators I used are
```
const goToPodmanDesktopButton = page.locator('button:text("Go to Podman Desktop")');
const dashboardTitle = page.getByRole('heading', { name: 'Dashboard' });
```

which are fare more robust. So we should grab real objects/aria labels, not use css objects that are too fragile

The same for the elements, I guess there are roles available (or we need to add them if missing)

Reply:
I totally agree, I wanted to have examples to continue with in case such option is not available. Roles are not yet everywhere, that is true, I will be discovering them while writing more tests.